internal/servicedeployer/terraform.go

```diff
@@ -149,7 +149,7 @@ func (tsd TerraformServiceDeployer) SetUp(ctx context.Context, svcInfo ServiceIn
 	}
 	err = p.Up(ctx, opts)
 	if err != nil {
-		return nil, fmt.Errorf("could not boot up service using Docker Compose: %w", err)
+		return &service, fmt.Errorf("could not boot up service using Docker Compose: %w", err)
```

Comment:
Maybe these changes could be done in a different PR.

I was thinking that it would be interesting to return also the service here in case of error, to be able to run the `shutdownServiceHandler`.

The same if there is any error while waiting to be healthy the container.

WDYT ?

Reply:
Removed for now these changes.